### DISCUSSION TITLE
Installing wasm bundle along with the rest of the npm package

### DISCUSSION THREAD

**Discussion by tjomson** [duckdb/duckdb#10511] [Q&A]
Hi there, I am new to duckdb wasm.
As far as I can tell, the wasm bundle is installed on runtime, the first time that the package is used.
I would like to use it in a package that should work offline, so I would like the wasm bundle to be included when the package is installed from npm. 
Is this possible?
Any answers are appreciated :)

**Comment by carlopi** [comment]
You are right with having this concern, but I don't think this is [completely] accurate, given indeed the main .wasm files are part of the npm package (see .wasm in this list: https://www.npmjs.com/package/@duckdb/duckdb-wasm?activeTab=code).

It depends how you are instantiating duckdb-wasm, are you using the JsDeliveryBundles (those are fetched online) or providing the npm paths?

This is something we should improve on, potentially changing defaults or changing docs, so suggestions are very welcome.

**Reply by tjomson** [reply]
Thanks for the response!
I made a new thread here: https://github.com/duckdb/duckdb-wasm/discussions/1628 
I was using the jsdelivr way described in the readme. I tried defining the paths like in the webpack example, but I can't use import.meta.url as i use commonjs

**Comment by carlopi** [comment]
[ Also noticing this is about duckdb-wasm, that has it's repo at https://github.com/duckdb/duckdb-wasm, further conversations are better had there! ]
